client/post-editor/media-modal/image-editor/image-editor-toolbar.jsx

```diff
@@ -70,12 +103,24 @@ const MediaModalImageEditorToolbar = React.createClass( {
 		return (
 			<div className="editor-media-modal-image-editor__toolbar">
 				{ this.renderButtons() }
+				<PopoverMenu isVisible={ this.state.showAspectPopover }
+						onClose={ this.onAspectClose }
+						position="top"
+						context={ this.refs && this.refs.aspectButton }
```

Comment:
Aside: This is common practice in our codebase, but the React docs strongly discourage accessing `refs` from render:

>_Never_ access refs inside of any component's render method – or while any component's render method is even running anywhere in the call stack.

https://facebook.github.io/react/docs/more-about-refs.html#cautions

Alternative would probably be something assigning the ref to the instance using [the callback attribute](https://facebook.github.io/react/docs/more-about-refs.html#the-ref-callback-attribute) and passing `context` via this value.